vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -324,7 +325,11 @@ def pad_input_ids(
         # this is a causal mask for generation
         mask = (mask.unsqueeze(-1) == mask.unsqueeze(-2)).tril()
         mask = torch.where(mask.logical_not(), -torch.inf, 0.0)
-        mask = mask.to(self.model.model.dtype)
+        # Currently, using this quantization will generate NaN, keep it float32
```

Comment:
this is still suspicious behavior right? I'm investigating this issue separately